templates/root.lucius

```diff
@@ -0,0 +1,28 @@
+@linkcolor: #E74C3D;
+@linkbordercolor: #F7C4BF;
+@lightgray: #D7D7D7;
+@mediumgray: #615E5F;
+@darkgray: #221E1F;
```

Comment:
Right now I've just duplicated these variables in the lucius files that need (any of) them. I've opened https://github.com/yesodweb/shakespeare/issues/152 to look into the best way to reduce that duplication.